### DISCUSSION TITLE
External light

### DISCUSSION THREAD

**Discussion by dan265** [magknight/787aviatorsTracker#534] [Completed things] [answered]
Hello. I would like to ask if its possible to increase light intensity expecially taxi light/rnw turnoff and landing light. Thanks

**Comment by aeroniemi** [comment, marked as answer]
They've already been increased in brightness in the alpha channel - they're
at a pretty realistic level now!

**Reply by dan265** [reply]
oh Nice, didn't notice on last flight because was in daytime. :)
